src/components/usa-banner/banner.stories.js

```diff
@@ -32,33 +54,21 @@ export const CustomContent = {
   },
   args: {
     label: "Un site Web officiel du gouvernement américain",
-    bannerText: "Un site Web officiel du gouvernement américain",
-    bannerAction: "Voici comment vous le savez",
-    domainHeading: "Les sites Web officiels utilisent",
-    domainText:
+    /**
+     * The `getStorybookHelpers` function from @wc-toolkit/storybook-helpers`
+     * automatically appends the `slot` attribute to avoid collisions with other props.
+     *
+     * To set the content for this story, the key should have the suffix `-slot`,
+     * even though the slot name in the component is `banner-text`, `banner-action`, etc.
+     */
+    "banner-text-slot": "Un site Web officiel du gouvernement américain",
+    "banner-action-slot": "Voici comment vous le savez",
+    "domain-heading-slot": "Les sites Web officiels utilisent .gov",
+    "domain-text-slot":
       "Un site Web .gov appartient à une organisation gouvernementale officielle aux États-Unis.",
-    httpsHeading: "Les sites Web .gov sécurisés utilisent HTTPS",
-    httpsText: `Un <strong>verrou</strong> (<span class="usa-banner__icon-lock" role="img" aria-label="Locked padlock icon"></span>) ou <strong>https://</strong> signifie que vous êtes connecté(e) en toute sécurité au site Web .gov. Assurez-vous de ne partager des informations sensibles que sur des sites Web officiels et sécurisés.`,
-    tld: "mil",
+    "https-heading-slot": "Les sites Web .gov sécurisés utilisent HTTPS",
+    "https-text-slot": `Un <strong>verrou</strong> ( <span class="usa-banner__icon-lock" role="img" aria-label="Locked padlock icon"></span> ) ou <strong>https://</strong> signifie que vous êtes connecté(e) en toute sécurité au site Web .gov. Assurez-vous de ne partager des informations sensibles que sur des sites Web officiels et sécurisés.`,
```

Comment:
I'm not doing a full review, but I noticed that the `aria-label` for the lock icon on this line is in English still, fyi 👀